tracks/bash/exercises/hamming/mentoring.md

```diff
@@ -91,19 +93,17 @@ The `&&` and `||` in bash are short-circuit logical operators. They stop evaluat
 
 `a && b` will only run b if a is true. `a || b` will only run b if a is false. `a || b || c` will stop as soon as anything is true. `a && b || c` will run a and ... b or c or both!! This is why a full blown if-else is often a good idea.
 
-* `if`/`else` used instead of List Constructs.
-
-Because conditionals are needed. If the student uses `if`/`else` it's an opportunity to talk about the list constructs for a more concise answer. `&&`/`||` The and list and or list constructs provide a means of processing a number of commands consecutively. These can effectively replace complex nested `if`/`then`  or even case statements. [List Constructs](https://tldp.org/LDP/abs/html/list-cons.html#LISTCONSREF)
-
 ### General Guidelines
```

Comment:
Are we not using new lines after headings in general? Or is this an exception?

Reply:
Added newlines after all headings.